README.md

```diff
@@ -22,9 +22,16 @@ Installation
 Running the site locally
 ------------------------
 
-1. `docker-compose down -v && docker-compose up --build`
+1. `docker-compose down -v  --remove-orphans && docker-compose up --build`
 2. Open `http://localhost:8080` in your browser.
 
+Changing parameters
+------------------------
+When running the site locally via Docker, the parameters are supplied by `/.docker/parameters.yaml`.
+
+To change a configuration that is supplied by an environment variable, edit `/.env`, or pass in as an environment variable at start up. For example, to change the API URL:
```

Comment:
Probably shouldn't promote editing `.env` as it's something committed. Maybe should be mentioned as reference to what can be changed?